uisdk/src/main/java/com/karhoo/uisdk/screen/booking/checkout/loyalty/LoyaltyFeatureFlags.kt

```diff
@@ -0,0 +1,7 @@
+package com.karhoo.uisdk.screen.booking.checkout.loyalty
+
+object LoyaltyFeatureFlags {
```

Comment:
The goal of the ticket is to unify feature flags API, so any plain scene in the code (like presenter) do ask for feature flag state using one storage. In the proposed solution, the problem of multiple sources of feature flags remain.